### PR TITLE
Fixes for linter action

### PR DESCRIPTION
**Pull request by rem1776**
**Description**
Updates the linter so it'll work correctly and cleans up the code so that it'll pass.

Most files had lines over the max length of 120 (it's really 119 since `\n`''s are counted) so the majority of the changes are line length fixes done mostly with a script.

**How Has This Been Tested?**
passed make distcheck with gcc 9 and intel 19

**Checklist:**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] New check tests, if applicable, are included
- [x] `make distcheck` passes

### CODE REVIEW

**Review by wrongkindofdoctor — APPROVED** [review]

**Comment by thomas-robinson** [comment]
This needs to be tested with a model before I will approve it.  There is a LOT of code changes.  Can you run the aquaplanet with this branch?

**Comment by rem1776** [comment]
@thomas-robinson Tested it with aquaplanet, no issues

File: time_manager/get_cal_time.F90 [hunk 1]
```diff
@@ -316,13 +317,15 @@ function get_cal_time(time_increment, units, calendar, permit_calendar_conversio
   increment_seconds = int(dt - increment_days*86400)
 else
   call error_mesg('get_cal_time','"'//trim(units)//'"'//' is not an acceptable units attribute of time.'// &
-    ' It must begin with: "years since", "months since", "days since", "hours since", "minutes since", or "seconds since"',FATAL)
+    ' It must begin with: "years since", "months since", "days since", "hours since", "minutes since", &
+            &  or "seconds since"',FATAL)
```
Comment (thomas-robinson) [inline time_manager/get_cal_time.F90:321]: This is inconsistent with how you have dealt with other strings in multi-line instances

**Review by thomas-robinson — APPROVED** [review]
There are inconsistencies with spacing and &s, but nothing that I see that is bad.  I made a few comments, but I don't think anything really needs to be addresed.

File: time_manager/get_cal_time.F90 (hunk 1 above)
Comment (rem1776) [reply time_manager/get_cal_time.F90]: The script got messed up with this line actually so this is fixed to add the concatenation

**Review by thomas-robinson — APPROVED** [review]

**Pull request merged**